samples/interactive-tutorials/README.md

```diff
@@ -16,34 +33,59 @@ After the project is created, set your PROJECT_ID to a ```project``` variable.
 
 1. Check that the Retail API is enabled for your Project in the [Admin Console](https://console.cloud.google.com/ai/retail/).
 
-### Set up authentication
+### Create service account
+
+To access the Retail API you must create a service account.
 
-To run a code sample from the Cloud Shell, you need to authenticate. To do this, use the Application Default Credentials.
+1. To create service account follow this [instruction](https://cloud.google.com/retail/docs/setting-up#service-account)
 
-1. Set your user credentials to authenticate your requests to the Retail API
+1. Find your service account on the [IAM page](https://console.cloud.google.com/iam-admin/iam),
+   click `Edit` icon and add the roles 'Storage Admin' and 'BigQuery Admin. It may take a while for the changes to take effect.
 
+1. Copy the service account email in the field Principal.
+
+### Set up authentication
+
+To run a code sample from the Cloud Shell, you need to be authenticated using the service account credentials.
+
+1. Login with your user credentials.
     ```bash
-    gcloud auth application-default login
+    gcloud auth login
     ```
 
 1. Type `Y` and press **Enter**. Click the link in Terminal. A browser window should appear asking you to log in using your Gmail account.
 
 1. Provide the Google Auth Library with access to your credentials and paste the code from the browser to the Terminal.
```

Comment:
2. Provide the Google Auth Library with access to your credentials and paste the code from the browser to the Terminal.